intake/utils.py

```diff
@@ -0,0 +1,10 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2018, Anaconda, Inc. and Intake contributors
+# All rights reserved.
+#
+# The full license is in the LICENSE file, distributed with this software.
+#-----------------------------------------------------------------------------
+
+def make_path_posix(path):
+    return path.replace('\\', '/')
```

Comment:
This is a pretty important function, should have tests of its own. I wonder if we should only do this when `os.sep == '\\'`?

Reply:
Definitely should have its own tests. I am not quite sure whether or not it should only be applied base on the os.sep. I imagine that depends on whether or not the user is allowed to use windows style paths in the catalog.

Reply:
Ah, you something like the case where a catalog is written on a windows machine and transferred to a linux one? I guess there's no way that the paths can make sense in such a situation anyway, so I would only apply in the windows-specific case.

Reply:
They might make sense if CATALOG_DIR is used.

Reply:
You have a point. OK, we can keep it as it is and see if further problems arise down the road.